drivers/unix/os_unix.cpp

```diff
@@ -699,6 +699,16 @@ Error OS_Unix::set_cwd(const String &p_cwd) {
 
 	return OK;
 }
+String OS_Unix::get_cwd()
+{
+	String cwd = "";
+	char ret[2048];
+	ERR_FAIL_NULL_V(getcwd(ret, 2048), ERR_BUG);
```

Comment:
```suggestion
	char ret[PATH_MAX];
	ERR_FAIL_NULL_V(getcwd(ret, PATH_MAX), ERR_BUG);
```

On some systems, `PATH_MAX` can be 4K.